QRCoder/BitmapByteQRCode.cs

```diff
@@ -53,13 +53,35 @@ public byte[] GetGraphic(int pixelsPerModule, byte[] darkColorRgb, byte[] lightC
     {
         var sideLength = QrCodeData.ModuleMatrix.Count * pixelsPerModule;
 
-        var moduleDark = darkColorRgb.Reverse();
-        var moduleLight = lightColorRgb.Reverse();
+        var moduleDarkSingle = darkColorRgb.Reverse().ToArray();
+        var moduleLightSingle = lightColorRgb.Reverse().ToArray();
```

Comment:
Interestingly, simply adding `.ToArray()` here probably was the single biggest factor affecting the benchmarks.